lib/pf/Authentication/Source/KerberosSource.pm

```diff
@@ -20,8 +20,9 @@ extends 'pf::Authentication::Source';
 
 has '+type' => ( default => 'Kerberos' );
 has 'host' => (isa => 'Str', is => 'rw', required => 1);
-has 'realm' => (isa => 'Str', is => 'rw', required => 1);
+has 'authenticate_realm' => (isa => 'Str', is => 'rw', required => 1);
 has 'stripped_user_name' => (isa => 'Str', is => 'rw', default => 'yes');
+has 'realm' => (isa => 'ArrayRef[Str]', is => 'rw');
```

Comment:
Since that field seems common to all sources, it should be moved in Authentication/Source.pm